### PR TITLE
Evaluate stale getter deps only once

### PR DESCRIPTION
**Pull request by loganlinn**
Move logic around to avoid evaluating dependencies twice in certain code path

### CODE REVIEW

File: src/evaluator.js [hunk 1]
```diff
@@ -53,22 +53,22 @@ class Evaluator {
...
 
-    } else if (this.__hasStaleValue(state, keyPathOrGetter)) {
+    }
+
+    // evaluate dependencies
+    var args = getDeps(keyPathOrGetter).map(dep => this.evaluate(state, dep))
+
+    if (this.__hasStaleValue(state, keyPathOrGetter)) {
+      // getter deps could still be unchanged since we only looked at the unwrapped (keypath, bottom level) deps
       var prevValue = this.__cachedGetters.getIn([code, 'value'])
```
Comment (jordangarcia) [inline src/evaluator.js]: pretty minor, but this could be put inside the `isEqual` if block

Comment (loganlinn) [inline src/evaluator.js]: loganlinn/nuclear-js@aeeb00d

**Comment by jordangarcia** [comment]
This is causing an evaluator test to fail.  Will look into it.

**Comment by jordangarcia** [comment]
@loganlinn would you mind merging master so that CI will run properly.

**Pull request merged**